Source/Core/Core/AchievementManager.cpp

```diff
@@ -1205,7 +1283,12 @@ void AchievementManager::HandleLeaderboardTriggeredEvent(const rc_runtime_event_
                                     m_game_data.leaderboards[ix].title),
                         OSD::Duration::VERY_LONG, OSD::Color::YELLOW);
       }
-      return;
+      m_queue.EmplaceItem([this, runtime_event] {
```

Comment:
Still wrong. `runtime_event` points to an object that likely may longer exist once the queue gets to this item, pass `event_id` here.